BeeSwift/AppDelegate.swift

```diff
@@ -41,87 +42,70 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
             UserDefaults.standard.set(true, forKey: Constants.healthSyncRemindersPreferenceKey)
         }
         
+        UNUserNotificationCenter.current().delegate = self
         NotificationCenter.default.addObserver(self, selector: #selector(self.updateBadgeCount), name: NSNotification.Name(rawValue: CurrentUserManager.goalsFetchedNotificationName), object: nil)
-
+        
         return true
     }
-
-    func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable : Any]) {
```

Comment:
removed because we use the iOS 10 UNUserNotificationCenterDelegate method instead